src/NuGet.Clients/NuGet.PackageManagement.UI/Options/AddMappingDialog.xaml

```diff
@@ -11,14 +11,11 @@
              x:Uid="AddMappingDialogWindow"
              Height="348"
              Width="480"
+             ResizeMode="NoResize"
```

Comment:
This one is intuitive. I did consider allowing resize. Most dialogs do not, but some do. I feel that the scrollbar is sufficient, for now, and didn't want to enable new behavior while trying to fix this issue. One concern is that there may be an expectation that window size is remembered, and I believe that would require some extra work to make that happen.

Reply:
Thinking about it... maybe now is the time to enable it. If customers want it to remember width, we can address that when it comes up.
Here's the commit to enable it. Maybe I should do a separate PR since it's scope creep? https://github.com/NuGet/NuGet.Client/commit/d8615a5ea6640dcf3ac2a4012244466488d60092
![image](https://user-images.githubusercontent.com/49205731/214352120-b26ae0ee-0872-4dbe-a2cf-8ba8284efc15.png)

Reply:
Draft PR waiting on this PR: https://github.com/NuGet/NuGet.Client/pull/5029